contractsSol5/KyberNetworkConnector.sol

```diff
@@ -0,0 +1,108 @@
+pragma solidity 0.5.11;
+
+import "./utils/Withdrawable3.sol";
+import "./utils/Utils4.sol";
+import "./utils/zeppelin/ReentrancyGuard.sol";
+import "./utils/zeppelin/SafeERC20.sol";
+import "./IKyberNetwork.sol";
+
+
+/*
+ *   @title Kyber Network Connector main contract
+ *   Interacts with contracts:
+ *       KyberNetwork: to call  fee data
+ *       KyberNetworkProxy: only proxy can call tradeWithHint
+ *
+ *    Kyber Network Connector will call KyberNetwork for:
+ *       - getExpectedRateWithHintAndFee
+ *       - tradeWithHintAndFee
+ */
+
+contract KyberNetworkConnector is
+    Withdrawable3,
```

Comment:
please have this withdrawable2.

we will use version 3 only when large contract issue